settings/settings.go

```diff
@@ -59,13 +59,13 @@ func Load() (*Settings, error) {
 
 	c, err := deis.New(sF.VerifySSL, sF.Controller, sF.Token)
 
-	// Set a custom user agent
-	c.UserAgent = UserAgent
-
 	if err != nil {
 		return nil, err
 	}
 
+	// Set a custom user agent
+	c.UserAgent = UserAgent
```

Comment:
got it. ok, I'm fine with both being in at once. can you make sure to add a short description regarding what this does (something like "prevent a panic if there's no user agent" iirc) in the final commit to master?